packages/core/core/src/applyRuntimes.js

```diff
@@ -216,6 +216,11 @@ export default async function applyRuntimes({
 
     if (isEntry) {
       bundleGraph._graph.addEdge(bundleNodeId, bundleGraphRuntimeNodeId);
+      // add entry to the front, remove previous runtime
```

Comment:
I noticed that entryAssetIds was growing since it was reusing the same bundle graph. Now it removes the previous entryAssetId and then adds it to the front.